libraries/c_sdk/standard/ble/src/services/wifi_provisioning/iot_ble_wifi_provisioning.c

```diff
@@ -1426,15 +1426,21 @@ void _listNetworkTask( IotTaskPool_t taskPool,
 
     status = WIFI_Scan( scanNetworks, wifiProvisioning.listNetworkRequest.maxNetworks );
 
+    uint32_t networks_found = 0;
```

Comment:
As per the coding standard, variable declaration needs to be at the beginning of the function.

Reply:
Got it.